ci/run-test-labshell.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env labshell
+#!/usr/bin/env bash
+#set -o xtrace
+
+ROOT_DIR=$(cd "$(dirname "$0")/.." && pwd)
+
+HW="$(find "$ROOT_DIR/" -mindepth 1 -maxdepth 1 -type d -name 'hw*' | sort | tail -n 1 | sed -nE 's/.*\/hw([0-9]+)$/\1/p')"
```

Comment:
You could simply use `$(ls $ROOT_DIR/hw* | ...)` instead.

Verification: In case this is run from master it will run the lexicographical last one, correct?

Should we run all of them maybe?

Reply:
The background of this is to be able to verify at once if the environment works, in case we have a solution that is known to be correct.

Reply:
> You could simply use `$(ls $ROOT_DIR/hw* | ...)` instead.

The use of `find` over `ls` is suggested for use in scripts (see https://github.com/koalaman/shellcheck/wiki/SC2012).

> Verification: In case this is run from master it will run the lexicographical last one, correct?

Yes.

> Should we run all of them maybe?

Maybe. But that's another issue.